docs/basics/rankine_cycle.rst

```diff
@@ -81,6 +81,39 @@ water return flow temperature.
 After rerunning, we will see that the condensation temperature and pressure
 are both automatically calculated by the specified terminal temperature value.
 
+Generating T-s Diagram
+^^^^^^^^^^^^^^^^^^^^^^
+To visualize the Rankine Cycle, we may generate a temperature (T) versus 
+entropy (s) diagram. To do this, we can import fluprodia (Fluid Property Diagram) 
+package. To successfully plot the T-s diagram, following steps are required:
```

Comment:
I feel like, right after mentioning fluprodia package, we show the code first, then explain the steps. Looks smoother in terms of flow to me. If you are happy with this change:

```suggestion
package.

.. dropdown:: Click to expand to code section

    .. literalinclude:: /../tutorial/basics/rankine.py
        :language: python
        :start-after: [sec_Ts]
        :end-before: [sec_5]
```